utils/twitch_online.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import json
+
+
+def is_twitch_online(client, secret, user):
+    '''
+        Checks whether a specific twitch account is currently live.
+
+        Parameters
+        ----------
+        client : int
+        Twitch API parameter.
+        secret : int
+        Twitch API parameter.
+        user : int
+        Username of twitch account to check.
+
+        Returns
+        ----------
+            boolean: whether the twitch account is currently live
+
+    '''
+    try:
+        
+        #twitch api parameters
+        TWITCHCLIENT_ID = client
+        TWITCHSECRET = secret
+
+        #twitch username, e.g. https://twitch.tv/Kiver would be Kiver
+        USERSTREAM = user
+
+
+        # URL to request OAuth Token
+        tokenurl = 'https://id.twitch.tv/oauth2/token?client_id=' + TWITCHCLIENT_ID + \
+                   '&client_secret=' + TWITCHSECRET+'&grant_type=client_credentials'
+
+
+        response = requests.post(tokenurl)
+        print("No bug yet")
+        response.raise_for_status()
+        OAuth_Token = response.json()["access_token"]
```

Comment:
This is quite a nit but I'd still like to see this variable in `this_case` i.e. `oauth_token`. Otherwise GitHub highlights it as a class or something :cry: